sdk/src/main/java/com/hedera/hashgraph/sdk/Client.java

```diff
@@ -291,6 +295,19 @@ public synchronized Client setNetwork(Map<String, AccountId> network) throws Int
         return network;
     }
 
+    public Void ping(AccountId nodeAccountId) {
```

Comment:
The reason we need to return `Void` here is because the originally generated method returned that. Since we override that method we need to maintain return types. If we were to remove those methods from being generated then I think that'd be considered a breaking change.